src/compas_rhino/geometry/curves/__init__.py

```diff
@@ -0,0 +1,28 @@
+from .nurbs import RhinoNurbsCurve
+
+try:
+    from compas.geometry import NurbsCurve
+except ImportError:
+    pass
+else:
+    from compas.plugins import plugin
+
+    @plugin(category='factories', requires=['Rhino'])
+    def new_nurbscurve(cls, *args, **kwargs):
+        return super(NurbsCurve, RhinoNurbsCurve).__new__(RhinoNurbsCurve)
```

Comment:
Is this one really correct with a `super` call here?

Reply:
i think so, but i will check